core/tests/protractor/editorAndPlayer.js

```diff
@@ -141,6 +141,7 @@ describe('Full exploration editor', function() {
         expectAvailableDestinationsToBe(['second', 'final card']);
       editor.addResponse('NumericInput', null, 'final card', false,
         'IsGreaterThan', 2);
+      // eslint-disable-next-line dot-notation
```

Comment:
I believe it is because ESLint is monitoring conflicts with reserved keywords. The following is what it gives as the error message: `.delete is a syntax error. (dot-notation)`